Command/SystemInfoDumpCommand.php

```diff
@@ -8,25 +8,55 @@
  */
 namespace EzSystems\EzSupportToolsBundle\Command;
 
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector;
```

Comment:
`registry_collector` was a mistake. `collector_registry` was the intention, good catch :-)